### PR TITLE
Add link to training, clarify "Need Help" section

### PR DESCRIPTION
**Pull request by tim-o**
Made "Need help" consistent with the [github template PR](https://github.com/cockroachdb/cockroach/pull/27811), and added references to the 2.0.4 training. @jseldess - are you OK with us pushing folks to the material?

If there's anyone else who should review a change to the readme, let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +45,11 @@ CockroachDB is production-ready. See our
 
 For guidance on installation, development, deployment, and administration, see our [User Documentation](https://cockroachlabs.com/docs/stable/).
 
+## Training
+
+Our [self guided training](https://www.cockroachlabs.com/docs/stable/training/) will show you how to install CRDB, import
```
Comment (jseldess) [inline README.md]: nit: `self guided training` > `self-guided training`

**Review by jseldess — APPROVED** [review]
LGTM, with a few nits. Thanks, @tim-o!

File: README.md [hunk 2]
```diff
@@ -77,19 +83,23 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
 
 ## Need Help?
 
-- [Troubleshooting documentation](https://www.cockroachlabs.com/docs/stable/troubleshooting-overview.html) -
+- If this is your first time using CRDB, we suggest completing [training before building your first cluster](https://www.cockroachlabs.com/docs/stable/training/). 
+  It's the same training we use internally. It covers everything from installing and importing your data to troubleshooting 
+  and decommissioning nodes, as well as the most common issues that trip up first time users.
```
Comment (jseldess) [inline README.md]: nit: `first time` > `first-time`

File: README.md [hunk 3]
```diff
@@ -77,19 +83,23 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
...
+  
+- [Check out our troubleshooting documentation](https://www.cockroachlabs.com/docs/stable/troubleshooting-overview.html) -
   Learn how to troubleshoot common errors, cluster and node setup, and SQL query behavior,
   and how to use debug and error logs.
 
-- [CockroachDB Forum](https://forum.cockroachlabs.com/) and
-  [Stack Overflow](https://stackoverflow.com/questions/tagged/cockroachdb) - Ask questions,
-  find answers, and help other users.
+- If that doesn't help and you're stuck or have a question, [contact us on the CockroachDB Forum (https://forum.cockroachlabs.com/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/cockroachdb).
```
Comment (jseldess) [inline README.md]: Missing the closing bracket after `Forum`.

File: README.md [hunk 4]
```diff
@@ -77,19 +83,23 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
...
-- [CockroachDB Forum](https://forum.cockroachlabs.com/) and
-  [Stack Overflow](https://stackoverflow.com/questions/tagged/cockroachdb) - Ask questions,
-  find answers, and help other users.
+- If that doesn't help and you're stuck or have a question, [contact us on the CockroachDB Forum (https://forum.cockroachlabs.com/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/cockroachdb). 
+  Our support team monitors the forum first for new questions.
+
+- If you've found a bug, or want to request a new feature, help us out by
+  [opening a Github issue](https://github.com/cockroachdb/cockroach/issues/new).
+
+- [We're also on Gitter](https://gitter.im/cockroachdb/cockroach)
```
Comment (jseldess) [inline README.md]: Missing period at end of sentence.

**Comment by jseldess** [comment]
@tim-o, can you address my comments before merging?

**Comment by tim-o** [comment]
Of course, I thought that was taken care of - I'm sorry, this is like PR #2, I'm still not super familiar with github.

**Comment by tim-o** [comment]
@jseldess - updated 
 here: https://github.com/cockroachdb/cockroach/pull/27813/commits/94399e6f2db9962f5ba57be5027e540dc917636e.

**Review by jseldess — APPROVED** [review]
LGTM. Thanks, @tim-o.

**Comment by jseldess** [comment]
@tim-o, do you still want to merge this?

**Comment by jseldess** [comment]
@tim-o, ping.

**Comment by tim-o** [comment]
@jseldess Removed conflicts, just waiting on Borse and CI and I'll merge.

**Comment by jordanlewis** [comment]
@jseldess do you still want this, should we close?

**Comment by jseldess** [comment]
@jordanlewis, I think we should close this at this point, as some of it is no longer accurate. I'd love to open a new PR with recommended readme updates, though.

**Pull request closed without merging**